bundles/org.openhab.binding.magentatv/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,238 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="magentatv"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="receiver">
+		<label>MagentaTV Media Receiver</label>
+		<description>Represents a Telekom Media Receiver for MagentaTV</description>
+
+		<channel-groups>
+			<channel-group id="control" typeId="control">
+				<label>Control</label>
+			</channel-group>
+			<channel-group id="program" typeId="program">
+				<label>Program Information</label>
+			</channel-group>
+			<channel-group id="status" typeId="status">
+				<label>Play Status</label>
+			</channel-group>
+		</channel-groups>
+
+
+		<config-description>
+			<parameter name="udn" type="text">
+				<label>Unique Device Name</label>
+				<description>The UDN identifies the Media Receiver</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="modelId" type="text">
+				<label>Receiver Model</label>
+				<description>Model type (MR4xx/ MR2xx)</description>
+				<required>true</required>
+				<options>
+					<option value="Auto">AUTO</option>
+					<option value="DMS_TPB">MR400</option>
+					<option value="MR401B">MR401B</option>
+					<option value="MR201">MR201</option>
+				</options>
+			</parameter>
+			<parameter name="ipAddress" type="text">
+				<label>Media Receiver's IP Address</label>
+				<description>IP Address of the receiver</description>
+				<required>true</required>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="port" type="text">
+				<label>Receiver Port</label>
+				<description>Port address for UPnP</description>
+				<required>true</required>
```

Comment:
Really? Isn't there a suitable default that you can assume if you already know the model?

Reply:
The port is discovered during the UPnP process

Reply:
What about textual configuration? The user should not be required to provide a port, if that is anyhow always the same for his receiver model.

Reply:
ok, implemented a default handling for MR400 and non-MR400 (all others)